src/multi_select.coffee

```diff
@@ -101,12 +114,33 @@ Ember.Widgets.MultiSelectComponent = Ember.Widgets.SelectComponent.extend
     @set 'values', Ember.A [] unless @get('values')
 
   deletePressed: (event) ->
-    if event.target.selectionStart == 0
+    if event.target.selectionStart is 0 and event.target.selectionEnd is 0
       @removeSelectItem(@get('selections.lastObject'))
+      event.preventDefault()
 
   removeSelectItem: (item) ->
+    # set the focus back to the searchView because this item will be removed
+    currentDropDownState = @get('showDropdown')
```

Comment:
another nitpick, but this is a weird name. Something like `dropdownIsShowing` would make line 125 read better.